lib/core/datatable/components/datatable/datatable.component.scss

```diff
@@ -390,8 +390,6 @@
         .ellipsis-cell {
             .cell-container {
                 height: 100%;
-                display: inline-grid;
-                width: 100%;
```

Comment:
does that mean cell content (including custom templates) cannot take full width anymore?